it/src/main/java/com/google/cloud/teleport/it/pubsublite/PubsubLiteResourceManager.java

```diff
@@ -0,0 +1,216 @@
+
+  /** Default implementation for {@link PubsubLiteResourceManager}. */
+  class DefaultPubsubliteResourceManager implements PubsubLiteResourceManager {
+
+    private final List<ReservationPath> cleanupReservations = new ArrayList<>();
+    private final List<TopicPath> cleanupTopics = new ArrayList<>();
+
+    @Override
+    public ReservationPath createReservation(
+        String reservationName, String cloudRegion, String projectId, Long capacity) {
+      try (AdminClient client =
+          AdminClient.create(
+              AdminClientSettings.newBuilder().setRegion(CloudRegion.of(cloudRegion)).build())) {
+        ReservationPath reservationPath =
+            ReservationPath.newBuilder()
+                .setProject(ProjectId.of(projectId))
+                .setLocation(CloudRegion.of(cloudRegion))
+                .setName(ReservationName.of(reservationName))
+                .build();
+        client
+            .createReservation(
+                Reservation.newBuilder()
+                    .setName(reservationPath.toString())
+                    .setThroughputCapacity(capacity)
+                    .build())
+            .get();
+        cleanupReservations.add(reservationPath);
+        return reservationPath;
+      } catch (ExecutionException | InterruptedException e) {
+        throw new RuntimeException(
+            String.format(
+                "Unable to create reservation %s in region %s with capacity %d",
+                reservationName, cloudRegion, capacity),
+            e);
+      }
+    }
+
+    @Override
+    public TopicName createTopic(String topicName, ReservationPath reservationPath) {
+      try (AdminClient client =
+          AdminClient.create(
+              AdminClientSettings.newBuilder().setRegion(reservationPath.location()).build())) {
+        TopicPath topicPath =
+            TopicPath.newBuilder()
+                .setName(TopicName.of(topicName))
+                .setLocation(reservationPath.location())
+                .setProject(reservationPath.project())
+                .build();
+        Topic topic =
+            client
+                .createTopic(
+                    Topic.newBuilder()
+                        .setName(topicPath.toString())
+                        .setPartitionConfig(
+                            Topic.PartitionConfig.newBuilder()
+                                .setCount(
+                                    100) // TODO(pabloem): Define a certain number of partitions(?)
+                                .build())
+                        .setRetentionConfig(
+                            Topic.RetentionConfig.newBuilder()
+                                // TODO(pableom): Define proper constant for period of retention.
+                                // 5 days
+                                .setPeriod(Duration.newBuilder().setSeconds(3600 * 24 * 5).build())
+                                .setPerPartitionBytes(30 * 1024 * 1024 * 1024L)
```

Comment:
nit: add constant, and a comment to explain?

Reply:
done

Reply:
one